paper-swatch-picker.html

```diff
@@ -183,6 +184,26 @@
         },
 
         /**
+         * The orientation against which to align the menu dropdown
+         * horizontally relative to the dropdown trigger.
+         */
+        horizontalAlign: {
```

Comment:
You need to bind these to the `paper-menu-button`, or else they won't do anything.